enforcer-rules/README.md

```diff
@@ -0,0 +1,88 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule verifies that the transitive dependency tree of `pom.xml` does not have
+any [linkage error](../library-best-practices/glossary.md#types-of-conflicts-and-compatibility).
+
+## Class path and dependencySection flag
+
+The rule takes `dependencySection` property to control the behavior (by default `DEPENDENCIES`).
```

Comment:
Maven properties are something else completely. https://maven.apache.org/pom.html#Properties

Perhaps rephrase as

The value of the dependencySection element determines whether the rule checks the dependencies in the dependencyManagement section or the dependencies section.

Reply:
Updated.
Yes, I see element is suitable word as in https://maven.apache.org/enforcer/maven-enforcer-plugin/usage.html